go1.10/main/Dockerfile.tmpl

```diff
@@ -12,6 +12,12 @@ RUN \
         gcc-multilib \
         libc6-dev \
         libc6-dev-i386 \
+        libpcap-dev \
```

Comment:
Packetbeat builds libpcap from source due to some issues we had to resolve:

- https://github.com/elastic/beats/blob/abae05768c4c445bddffd6fff520a62ccca2e126/packetbeat/magefile.go#L260-L262
- https://github.com/elastic/beats/pull/4217

I think you will have to do the same thing in the image if you fully want to make the image fully self-contained. Did you already test building Packetbeat with this image using the OS provided libpcap?

Addressing libpcap could be left until the end. It's not really a blocker for the issue you are trying to resolve. Having it in the image will speed up the builds, but we don't utilize any `apt` commands from Packetbeat's build to get libpcap built.

Reply:
I removed the `libpcap` libraries for now.  We can reassess it at the end when all the other packages are addressed.